app/policies/v2/report_policy.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V2
+  class ReportPolicy < V2::ApplicationPolicy
+    def index?
+      true # FIXME: this is handled in scoping
+    end
+
+    def show?
+      true # TODO: match_account?
```

Comment:
https://github.com/RedHatInsights/compliance-backend/blob/master/app/controllers/v2/application_controller.rb#L63-L66

and 

https://github.com/RedHatInsights/compliance-backend/blob/master/app/controllers/v2/policies_controller.rb#L76